transcom/mymove
# 20989 update duty_locations table data

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1002495&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## [INT PR](https://github.com/transcom/mymove/pull/13738)

## Summary

We received some updates to the transportation offices that should be marked true for provides_services_counseling from J9.  This will update the table in all environments.

### How to test

1. Access the milmove app as a customer.
2. Run the below GET API in Postman for one of the updated duty locations.
http://milmovelocal:3000/internal/duty_locations?search='F.E. Warren' (You will grab the "id" in the first return block.)
3.  Run the below GET API in Postman to pull back counseling offices associated to the above duty location.
http://milmovelocal:3000/internal/transportation_offices/:dutyLocationId/counseling_offices
4.  Verify that the new associated transportation office is returned in result set.
5. Also, can run the below sql in DBeaver to verify the migration ran and the DB is updated with the new data.

select dl.id as duty_location_id, dl.name as duty_location_name, transportation_office_id, tr.name as transportation_office_name, provides_services_counseling 
from duty_locations dl
join transportation_offices tr on tr.id = dl.transportation_office_id 
where transportation_office_id 
in (
'1b3e7496-efa7-48aa-ba22-b630d6fea98b',
'972c238d-89a0-4b50-a0cf-79995c3ed1e7',
'485ab35a-79e0-4db4-9f13-09f57532deee',
'a877a317-be5f-482b-a126-c91a34be9290',
'3210a533-19b8-4805-a564-7eb452afce10',
'f69f315c-942a-4ef3-9427-7fee7883ce73',
'1039d189-39ba-47d4-8ed7-c96304576862',
'e4a02d40-2ad9-44c5-a357-202d4ff0b51d',
'57cf1e81-8113-4a52-bc50-3cb8902c2efd'
)
